eos-extra-resize

```diff
@@ -32,9 +32,16 @@ if [ -z "$extra_disk" ]; then
     exit 0
 fi
 
+pt_label=$(blkid -o value -s PTTYPE $extra_disk)
+
+if [ "$pt_label" = "gpt" ]; then
+  marker=$(sfdisk --force --part-attrs $extra_disk 1)
+else
+  marker=$(sfdisk --force --part-type $extra_disk 4)
```

Comment:
As before we are using `printf "\xdd" | dd of=${disk1_img} bs=1 count=1 seek=498 conv=notrunc` in `eos-image-builder` to write a fake partition `4`